src/nvim/event/libuv_process.c

```diff
@@ -90,7 +90,7 @@ int libuv_process_spawn(LibuvProcess *uvproc)
 
   int status;
   if ((status = uv_spawn(&proc->loop->uv, &uvproc->uv, &uvproc->uvopts))) {
-    ELOG("uv_spawn(%s) failed: %s", uvproc->uvopts.file, uv_strerror(status));
+    DLOG("uv_spawn(%s) failed: %s", uvproc->uvopts.file, uv_strerror(status));
```

Comment:
Can this be `ILOG` ? Why would we want to ignore this error?

Reply:
Changed to `ILOG` in https://github.com/neovim/neovim/commit/ae28ef327e02ac87ef26f941c401312ed0462d8c.